src/views/RecIM/components/Forms/ActivityForm/index.jsx

```diff
@@ -193,10 +183,17 @@ const ActivityForm = ({ activity, closeWithSnackbar, openActivityForm, setOpenAc
 
     let activityRequest = { ...currentInfo, ...newInfo };
 
+    activityRequest.statusID = activityStatusTypes.find(
```

Comment:
The statusID is not needed for the creation of an activity. It is fine to have it here but it is technically only needed if the activity is being modified.